xcp_d/interfaces/prepostcleaning.py

```diff
@@ -51,9 +53,9 @@ class removeTR(SimpleInterface):
     output_spec = _removeTROutputSpec
 
     def _run_interface(self, runtime):
-
+        volumes_to_drop = np.ceil(self.inputs.time_todrop / self.inputs.TR)
```

Comment:
Let's keep this outside of this function so we don't calculate it twice. We have to calculate it for the boilerplate in the init_postprocess_wf